packages/multichain-account-service/src/MultichainAccountService.ts

```diff
@@ -39,6 +49,8 @@ export type MultichainAccountServiceOptions = {
   providerConfigs?: {
     [EvmAccountProvider.NAME]?: EvmAccountProviderConfig;
     [SolAccountProvider.NAME]?: SolAccountProviderConfig;
+    [BtcAccountProvider.NAME]?: BtcAccountProviderConfig;
+    [TrxAccountProvider.NAME]?: TrxAccountProviderConfig;
```

Comment:
Actually, we can remove those, those 2 providers are not yet considered "stable" thus, we cannot forward those configs to their providers yet (this we have to be passed in the clients service initialization directly)
```suggestion
```

Reply:
That being said... It's only about types here, so we can probably leave it as-is, that's fine actually! My bad 😁